From akpm@osdl.org Tue Apr 18 21:07:33 2006
Message-Id: <200604190407.k3J47TZa008130@shell0.pdx.osdl.net>
Subject: PCI: 64 bit resource: fix amba build warning
To: greg@kroah.com
Cc: akpm@osdl.org
From: akpm@osdl.org
Date: Tue, 18 Apr 2006 21:06:47 -0700


From: Andrew Morton <akpm@osdl.org>

drivers/amba/bus.c: In function `show_resource':
drivers/amba/bus.c:183: warning: long long unsigned int format, different type arg (arg 5)

Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/amba/bus.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/amba/bus.c
+++ gregkh-2.6/drivers/amba/bus.c
@@ -180,7 +180,7 @@ static DEVICE_ATTR(name, S_IRUGO, show_#
 amba_attr(id, "%08x\n", dev->periphid);
 amba_attr(irq0, "%u\n", dev->irq[0]);
 amba_attr(irq1, "%u\n", dev->irq[1]);
-amba_attr(resource, "\t%016llx\t%016llx\t%016llx\n",
+amba_attr(resource, "\t%016llx\t%016llx\t%016lx\n",
 	 (unsigned long long)dev->res.start, (unsigned long long)dev->res.end,
 	 dev->res.flags);
 
